src/ExportMap.js

```diff
@@ -513,14 +514,20 @@ ExportMap.parse = function (path, content, context) {
 
     // This doesn't declare anything, but changes what's being exported.
     if (n.type === 'TSExportAssignment') {
-      const d = ast.body.find(
-        b => b.type === 'TSModuleDeclaration' && b.id.name === n.expression.name
+      const md = ast.body.find(
```

Comment:
it might be nice if we used whole words for these var names, it’s hard to know what’s going on right now :-)